selfservice/strategy/oidc/strategy_login.go

```diff
@@ -138,7 +138,7 @@ func (s *Strategy) handleConflictingIdentity(ctx context.Context, w http.Respons
 		return ConflictingIdentityVerdictReject, nil, nil, nil
 	}
 
-	verdict = s.conflictingIdentityPolicy(existingIdentity, newIdentity)
+	verdict = s.conflictingIdentityPolicy(existingIdentity, newIdentity, provider, bool(claims.EmailVerified))
```

Comment:
How about we pass in the whole claims to the policy? Then the policy has more information and we don't have to extend the API in the future?

Reply:
Done.